src/obu.rs

```diff
@@ -137,6 +137,13 @@ unsafe fn parse_seq_hdr(
         return Err(EINVAL);
     }
 
+    const DEBUG_SEQ_HDR: bool = false;
```

Comment:
Is this value meant to be edited directly by a programmer when they're debugging, or will it eventually be hooked up to an env var or a compiler flag?

Reply:
I think the programmer is meant to edit it.  It's a #define DEBUG_SEQ_HDR false` in C.